docs/configuration.rst

```diff
@@ -295,7 +295,7 @@ below:
 
    DD_PROFILING_ENABLE_CODE_PROVENANCE:
      type: Boolean
-     default: False
+     default: True
```

Comment:
do we want a `version_added` note about when this changed?

Reply:
ah yes, good call